ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KVOffset.scala

```diff
@@ -35,13 +35,14 @@ object KVOffset {
     val middle = BigInt(second) << ((end - lowestStart) * 8)
     val lowest = BigInt(third)
     val bytes = (maxValuePlusOne | highest | middle | lowest).toByteArray.drop(1) // this retains leading zeros
-    Offset.fromBytes(bytes)
+    Offset.fromByteArray(bytes)
   }
 
   def highestIndex(offset: Offset): Long =
-    BigInt(offset.toByteArray.slice(highestStart, middleStart)).toLong
+    BigInt(offset.bytes.toByteArray.slice(highestStart, middleStart)).toLong
```

Comment:
calling `toByteArray` on `Offset` directly should work, right?

Reply:
fixed